bin/index.js

```diff
@@ -459,9 +461,10 @@ async function fromCmd(packageName, opts) {
     return null;
 }
 
-async function httpCmd(json = "nsecure-result.json") {
+async function httpCmd(json = "nsecure-result.json", { port }) {
     const dataFilePath = join(process.cwd(), json);
-    const httpServer = await startHTTPServer(dataFilePath);
+    const configPort = Number.isNaN(Number(port)) ? 0 : Number(port);
```

Comment:
@fraxken : I change to use `Number.isNaN`,  i'm not really comfortable with this function..
Let me know if it's sound good for you.